### PR TITLE
Support Verve MRAID creatives

### PR DESCRIPTION
**Pull request by faresbh**
- Remove `useRawResponse` from adcel response ( adding a label on the top of the creative)
- Differentiate between Verve and normal MRAID expandable by ad URL that contain "`tags-prod.vrvm.com`" and "`type=expandable`". 
- Verve MRAID expandable creative should expand using a special parameter for the `expand` function in `HyBidMRAIDView`.
- Adding test creatives (The folder will be removed after merge)

### CODE REVIEW

File: PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDUtil.m [hunk 1]
```diff
@@ -27,17 +27,27 @@ @implementation PNLiteMRAIDUtil
...
+                                               withTemplate:@""];
+//     Remove the mraid.js script tag.
+//     We expect the tag to look like this:
+//     <script src='mraid.js'></script>
+//     But we should also be to handle additional attributes and whitespace like this:
+//     <script  type = 'text/javascript'  src = 'mraid.js' > </script>
+//
+    pattern = @"<script\\s+[^>]*\\bsrc\\s*=\\s*([\\\"\\\'])mraid\\.js\\1[^>]*>\\s*</script>\\n*";
+
+    regex = [NSRegularExpression regularExpressionWithPattern:pattern
```
Comment (can-soykarafakili) [inline PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDUtil.m:50]: After this change PubNative ads are also works fine, along with the verve ads i think?

File: PubnativeLite/PubnativeLite/Ad Model/HyBidAd.m [hunk 2]
```diff
@@ -83,6 +83,16 @@ - (NSString *)htmlData {
...
+    
+//    NSString *path = [[NSBundle mainBundle] pathForResource:@"ad8" ofType:@"txt"];
+//    NSError *error;
+//    NSString *content = [NSString stringWithContentsOfFile:path encoding:NSUTF8StringEncoding error:&error];
+//
+//    if (error)
+//        NSLog(@"Error reading file: %@", error.localizedDescription);
+//
+//    NSLog(@"contents: %@", content);
+//    return content
```
Comment (can-soykarafakili) [inline PubnativeLite/PubnativeLite/Ad Model/HyBidAd.m:95]: Are those commented out lines necessary? If not, let's remove them 👍

Comment (faresbh) [reply PubnativeLite/PubnativeLite/Ad Model/HyBidAd.m:95]: Those are commented on purpose for test. I left also the test creative folder.

File: PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDUtil.m (hunk 1 above)
Comment (faresbh) [reply PubnativeLite/PubnativeLite/MRAID/PNLiteMRAIDUtil.m:50]: I didnt remove that part I just moved below and add a regex for `useRawResponse`

File: PubnativeLite/PubnativeLite/MRAID/HyBidMRAIDView.m [hunk 3]
```diff
@@ -577,19 +577,25 @@ - (void)expand:(NSString *)urlString {
         
         // Check to see whether we've been given an absolute or relative URL.
         // If it's relative, prepend the base URL.
-        urlString = [urlString stringByRemovingPercentEncoding];
-        if (![[NSURL URLWithString:urlString] scheme]) {
-            // relative URL
-            urlString = [[[baseURL absoluteString] stringByRemovingPercentEncoding] stringByAppendingString:urlString];
```
Comment (faresbh) [inline PubnativeLite/PubnativeLite/MRAID/HyBidMRAIDView.m:583]: @eros902002 here's the encoding

**Pull request merged**